lerna.json

```diff
@@ -1,7 +1,4 @@
 {
-  "packages": [
-    "packages/*",
-    "test-packages/*"
-  ],
+  "packages": ["packages/*", "test-packages/*"],
```

Comment:
Not really part of this change, but required for the Linter to pass. If you like the old format, this file could be added to the `.prettierignore` instead